packages/mobile/ios/Podfile.lock

```diff
@@ -1229,4 +1229,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 530d428d567e9f985838a426dfa80fe9dabad31f
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.12.0
```

Comment:
I hit the same thing. Which is the good version?

Reply:
the lower one is